ReactSkia/core_modules/Alert.h

```diff
@@ -0,0 +1,31 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+#pragma once
+
+namespace facebook {
+namespace react {
+
+class Alert {
```

Comment:
why is this class defined seperately ?

Reply:
We want to have a separate Alert class where we can keep alert-related Parameters in the same place and use it accordingly.

For Eg.:
title,
message,
callbackOrButtons,
type,
defaultValue,
keyboardType,
options